docker/download-deconz.sh

```diff
@@ -6,18 +6,17 @@ DECONZ_VERSION=$1
 CHANNEL=$2
 PLATFORM=$3
 
-if echo "${PLATFORM}" | grep -qE "arm64";
-then
-  URL="http://deconz.dresden-elektronik.de/debian/${CHANNEL}/deconz_${DECONZ_VERSION}-debian-buster-${CHANNEL}_arm64.deb"
-fi
-if echo "${PLATFORM}" | grep -qE "amd64";
-then
-  URL="http://deconz.dresden-elektronik.de/ubuntu/${CHANNEL}/deconz-${DECONZ_VERSION}-qt5.deb"
-fi
-if echo "${PLATFORM}" | grep -qE "v7";
-then
-  URL="http://deconz.dresden-elektronik.de/raspbian/${CHANNEL}/deconz-${DECONZ_VERSION}-qt5.deb"
-fi
+case "{$PLATFORM}" in
```

Comment:
I think there is just a typo here. It should be
```suggestion
case "${PLATFORM}" in
```